content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -9,7 +9,7 @@ weight: 70
 
 {{% capture overview %}}
 
-{{< feature-state for_k8s_version="1.11" state="beta" >}}
+{{< feature-state for_k8s_version="1.14" state="GA" >}}
 
 [Pods](/docs/user-guide/pods) can have _priority_. Priority indicates the
```

Comment:
Hello @bsalamat, would you update the state attribute to one of:
"alpha, beta, deprecated, stable"? Hopefully, this will fix the build.

Reply:
done! Thanks.